include/aws/io/retry_strategy.h

```diff
@@ -112,6 +112,8 @@ struct aws_exponential_backoff_retry_options {
     size_t max_retries;
     /** Scaling factor to add for the backoff. Default is 25ms */
     uint32_t backoff_scale_factor_ms;
+    /** Maximum retry backoff in seconds. Default is 20 seconds */
+    uint32_t maximum_backoff_secs;
```

Comment:
trivial: the existing `max_retries` already abbreviated "maximum"
```suggestion
    uint32_t max_backoff_secs;
```